Assuming unrestricted shared filesystem usage.
None
host: b2-004
Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Conda environments: ignored
Job stats:
job                count
---------------  -------
validate_inputs        1
total                  1

Select jobs to execute...
Execute 1 jobs...
[Mon May 19 16:30:55 2025]
localrule validate_inputs:
    input: /ddn_scratch/roles/strain_analysis/iHMP/patient_ids.yaml
    output: logs/validation_complete
    log: /ddn_scratch/roles/strain_analysis/iHMP/output/logs/validate_inputs.log
    jobid: 0
    reason: Missing output files: logs/validation_complete
    resources: tmpdir=/tmp

RuleException:
CalledProcessError in file "/ddn_scratch/roles/strain_analysis/iHMP/strainscape/snakemake/Snakefile", line 59:
Command 'set -euo pipefail;  python ../strainscape/validate_inputs.py config/config.yaml > /ddn_scratch/roles/strain_analysis/iHMP/output/logs/validate_inputs.log 2>&1' returned non-zero exit status 1.
[Mon May 19 16:30:55 2025]
Error in rule validate_inputs:
    message: None
    jobid: 0
    input: /ddn_scratch/roles/strain_analysis/iHMP/patient_ids.yaml
    output: logs/validation_complete
    log: /ddn_scratch/roles/strain_analysis/iHMP/output/logs/validate_inputs.log (check log file(s) for error details)
    shell:
        python ../strainscape/validate_inputs.py config/config.yaml > /ddn_scratch/roles/strain_analysis/iHMP/output/logs/validate_inputs.log 2>&1
        (command exited with non-zero exit code)
Shutting down, this might take some time.
Exiting because a job execution failed. Look below for error messages
[Mon May 19 16:30:55 2025]
Error in rule validate_inputs:
    message: None
    jobid: 0
    input: /ddn_scratch/roles/strain_analysis/iHMP/patient_ids.yaml
    output: logs/validation_complete
    log: /ddn_scratch/roles/strain_analysis/iHMP/output/logs/validate_inputs.log (check log file(s) for error details)
    shell:
        python ../strainscape/validate_inputs.py config/config.yaml > /ddn_scratch/roles/strain_analysis/iHMP/output/logs/validate_inputs.log 2>&1
        (command exited with non-zero exit code)
Complete log(s): /ddn_scratch/roles/strain_analysis/iHMP/strainscape/snakemake/.snakemake/log/2025-05-19T163055.526932.snakemake.log
WorkflowError:
At least one job did not complete successfully.
